Tests/SessionManagerTests.swift

```diff
@@ -131,7 +177,7 @@ class SessionManagerTestCase: BaseTestCase {
         XCTAssertNil(manager, "manager should be nil")
     }
 
-    // MARK: Start Requests Immediately Tests
+    // MARK: Tests - Start Requests Immediately
```

Comment:
This is just the convention I've used throughout the rest of the test suite. It helps more clearly define what are tests and what are helper types, properties, private helper methods, etc. I'd recommend we leave this as is for now then we can revisit the whole test suite once we ship 4.0.